### PR TITLE
Feat: add a ReScript runtime and a wrapper for `french_law.js`

### PR DESCRIPTION
**Pull request by EmileRolley**
Packages:
* a ReScript Catala runtime wrapper under [@catala-lang/rescript-catala](https://www.npmjs.com/package/@catala-lang/rescript-catala)
* a ReScript wrapper for `french_law.js` under [@catala-lang/french-law](https://www.npmjs.com/package/@catala-lang/french-law?activeTab=readme)

Related PR from the website: https://github.com/CatalaLang/catala-website/pull/52

### CODE REVIEW

File: french_law/js/package-lock.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+  "name": "@catala-lang/french-law",
+  "version": "0.5.0-a.1",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
-      "name": "french_law",
-      "version": "0.5.0",
+      "name": "@catala-lang/french-law",
+      "version": "0.5.0-a.1",
```
Comment (denismerigoux) [inline french_law/js/package-lock.json:9]: The version here should be bumped to match the version of Catala (0.8.0).

File: french_law/js/package.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "french_law",
-  "version": "0.5.0",
+  "name": "@catala-lang/french-law",
+  "version": "0.5.2",
```
Comment (denismerigoux) [inline french_law/js/package.json:3]: Same bump to 0.8.0

File: generate_website_assets.sh [hunk 3]
```diff
@@ -26,6 +26,9 @@ rsync $BUILD/catala.html $1/
 rsync $BUILD/clerk.html $1/
 rsync $BUILD/catala_legifrance.html $1/
 
-rsync $BUILD/french_law/js/french_law.js $1/french_law.js
+# NOTE(@EmileRolley): the french_law.js is accessible through:
+# https://www.npmjs.com/package/@catala-lang/french-law
+# no need to copy it here.
+# rsync $BUILD/french_law/js/french_law.js $1/french_law.js
```
Comment (denismerigoux) [inline generate_website_assets.sh:32]: You can just delete the line, no need to put the comment. However you should put a README section about how to generate new website assets and how to publish them on NPM.

File: runtimes/rescript/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+```
+# in this dir
+yarn link
+
+# in the project to use the lib
+yarn link @catala-lang/rescript-catala
+```
+
+> _Somehow it doesn't work with
```
Comment (denismerigoux) [inline runtimes/rescript/README.md:21]: ?

**Review by denismerigoux — APPROVED** [review]
Thanks for this high-quality PR. Just a few nitpickings and we can merge!

File: french_law/js/package.json (hunk 2 above)
Comment (EmileRolley) [reply french_law/js/package.json:3]: The frequency of updates may go faster for the french law, no?

Comment (denismerigoux) [reply french_law/js/package.json:3]: Yes, but let's keep the major (and possibly minor) versionning in sync with the version of Catala.

File: runtimes/rescript/README.md (hunk 4 above)
Comment (EmileRolley) [reply runtimes/rescript/README.md:21]: I didn't succeed to have the local package working correctly and I didn't want to loose time on it, as it should not require repetitive updates.

**Pull request merged**